js/kraken.js

```diff
@@ -57,7 +57,89 @@ module.exports = class kraken extends Exchange {
                     'https://www.kraken.com/en-us/help/api',
                     'https://github.com/nothingisdead/npm-kraken-api',
                 ],
-                'fees': 'https://www.kraken.com/en-us/help/fees',
+                'fees': [
+                    'https://www.kraken.com/en-us/help/fees',
+                    'https://support.kraken.com/hc/en-us/articles/201396777-What-are-the-deposit-fees-',
+                    'https://support.kraken.com/hc/en-us/articles/201893608-What-are-the-withdrawal-fees-',
+                ],
+            },
+            'fees': {
+                'trading': { 
+                    'tierBased': true,
+                    'percentage': true,
+                    'taker': [
+                        [0, 0.26 / 100],
```

Comment:
sorry, we can't do this like that, without changing the other functions that need those values, the taker and maker should be floats...